src/config/dynamic-theme-fixes.config

```diff
@@ -585,6 +585,27 @@ body[style] {
 
 ================================
 
+bestbuy.ca
+
+INVERT
+.centerContainer
+
+CSS
+[class^="facetName"], [class^="title"], [class^="sortLabel"], [class^="headerText"], [class^="subTitle"] {
```

Comment:
Where can I find this element?
And This is the same as `INVERT` so you can move the classes to that section

Reply:
https://www.bestbuy.ca/en-ca/category/gaming-monitors/474835
I understand that this is the same as `INVERT`, however the class names include a random code at the end. It is therefore require to list these in CSS with the operand to make sur all selectors that start with the selector names are selected, no matter the random code added at the end.

Is there are way to specify a search string in the class section, or must the class selectors in that section be whole?

Reply:
Yes, exactly.

Reply:
So you can just have all CSS selectors on a new line and it should work the same way.

```CSS
INVERT
[class^="facetName"]
[class^="title"]
[class^="sortLabel"]
[class^="headerText"]
[class^="subTitle"]
```
Is the same thing.

Reply:
As long as you add a comma (,) before each new line. Otherwise, the browser will think that this is one longer selector and none of it will work. Unless the DarkReader extension automatically adds a comma in such cases.

Reply:
> Unless the DarkReader extension automatically adds a comma in such cases.

We do. The example is a copy-paste ready code to be used.